src/libs/filters/rating.c

```diff
@@ -99,21 +99,22 @@ static gchar *_rating_print_func(const double value, const gboolean detailled)
         return g_strdup(_("rejected"));
       case 0:
         return g_strdup(_("not rated"));
-      case 1:
-        return g_strdup("★");
-      case 2:
-        return g_strdup("★ ★");
-      case 3:
-        return g_strdup("★ ★ ★");
-      case 4:
-        return g_strdup("★ ★ ★ ★");
-      case 5:
-        return g_strdup("★ ★ ★ ★ ★");
     }
   }
   return g_strdup_printf("%.0lf", floor(value));
 }
 
+static gchar *_rating_current_text_func(GtkDarktableRangeSelect *range, const double current)
+{
+  gchar *hovered = range->print(current, TRUE);
+  gchar *selected = g_markup_escape_text(dtgtk_range_select_get_rating_bounds_pretty(range), -1);
```

Comment:
we need another step here... (other we are leaking the string returned by `dtgtk_range_select_get_rating_bounds_pretty`)
something like : 
```
gchar *sel0 = dtgtk_range_select_get_rating_bounds_pretty(range);
gchar *selected = g_markup_escape_text(sel0, -1);
g_free(sel0);
```

Reply:
fixed